QuestionBot/QuestionBot/Model/QuestionMessageListener.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuestionBot.Model{
+
+    public class QuestionMessageListener : IMessageListener{
+
+        private IStore _questionDataStore;
+
+        public QuestionMessageListener(IStore store){
+            _questionDataStore = store;
+        }
+
+        public void ReceiveMessage(string input){
```

Comment:
Minor: The parameter name should match the parameter name on the interface (in this case, `message`)